fdbclient/include/fdbclient/TaskBucket.h

```diff
@@ -480,7 +480,8 @@ struct TaskFuncBase : IDispatched<TaskFuncBase, Standalone<StringRef>, std::func
 };
 #define REGISTER_TASKFUNC(TaskFunc) REGISTER_FACTORY(TaskFuncBase, TaskFunc, name)
 #define REGISTER_TASKFUNC_ALIAS(TaskFunc, Alias)                                                                       \
-	REGISTER_DISPATCHED_ALIAS(TaskFunc, Alias, TaskFunc::name, LiteralStringRef(#Alias))
+	REGISTER_DISPATCHED_ALIAS(                                                                                         \
+	    TaskFunc, Alias, TaskFunc::name, StringRef(reinterpret_cast<const uint8_t*>(#Alias), sizeof(#Alias) - 1))
```

Comment:
This is the only usage of LiteralStringRef that I couldn't directly remove and had to convert to a StringRef expression.

Reply:
I apparently got this wrong the first time (forgot to subtract 1), so maybe it's wise to still abstract this to a function. Perhaps if we wanted to do that, we could name it something else to remove the temptation to use `LiteralStringRef` where `_sr` would suffice? Or we could leave it like this. There are a small number of places where a line like above appears -- here, in a couple macros, and in a comparison operator.